modules/express-engine/README.md

```diff
@@ -60,6 +60,23 @@ app.engine('html', ngExpressEngine({
 }));
 ```
 
+## SSL
+
+SSL is enabeled by default. We provide a simple `addSSL` function inside of the `server-tools` folder. 
+In the `server.ts` is a `serverSettings` object where you can add the fully qualified paths to the certificates you want to use. When those are not provided, we will create a self-signed certificate. This will cause a warning in the browser.
```

Comment:
Explain that these self signed certs should only be used for development, and not production. 

If you explain that, then the last sentence `This will cause a warning` will make sense. Otherwise it is weird that we are admitting that we are adding somethign that will cause the browser to have a warning.